terraform/service_apiserver.tf

```diff
@@ -149,8 +149,19 @@ resource "google_cloud_run_service" "apiserver" {
 
   depends_on = [
     google_project_service.services["run.googleapis.com"],
+
     google_secret_manager_secret_iam_member.apiserver-db,
+    google_kms_key_ring_iam_member.kms-signerverifier,
```

Comment:
Is it possible to define this collection somewhere and be referenced here?

Reply:
That would certainly be nice, but no.

Reply:
I am consistently disappointed in aspects of this tool. Either we're doing something wrong, or there's an opportunity here.